qemu/tests/qemu_disk_img_commit.py

```diff
@@ -47,6 +48,8 @@ def run(test, params, env):
         raise error.TestError("Fail to save tmp file")
     commit_test.destroy_vm()
     commit_test.commit()
+    error.context("add sync operation on host after commit", logging.info)
```

Comment:
Hi xiagao,
Could you improve the description, it sound like an event not step 
e.g "sync host data after commit"